blank-canvas-blocks/sass/elements/_body.scss

```diff
@@ -0,0 +1,9 @@
+@import '../base/breakpoints';
+
+body {
+	padding: var(--wp--custom--margin--body-vertical) var(--wp--custom--margin--body-horizontal);
+
+	@include break-mobile {
+		padding: var(--wp--custom--margin--body-vertical-desktop) var(--wp--custom--margin--body-horizontal-desktop);
+	}
```

Comment:
Leveraging and then redefining the variables here would allow for other places that use the body padding variables to do so only once (the value of the variable would change based on the breakpoint)

```
body {
	padding-top: var(--wp--custom--body--padding--top);
	padding-bottom: var(--wp--custom--body--padding--bottom);
	padding-left: var(--wp--custom--body--padding--left);
	padding-right: var(--wp--custom--body--padding--right);

	@include break-mobile-only {
		--wp--custom--body--padding--top: var(--wp--custom--body--mobile--padding--top);		
		--wp--custom--body--padding--bottom: var(--wp--custom--body--mobile--padding--bottom);		
		--wp--custom--body--padding--left: var(--wp--custom--body--mobile--padding--left);		
		--wp--custom--body--padding--right: var(--wp--custom--body--mobile--padding--right);		
	}
}
```